test/tools/router_check/router.cc

```diff
@@ -396,6 +402,16 @@ bool RouterCheckTool::compareResults(const std::string& actual, const std::strin
   return false;
 }
 
+// The Mock for runtime value checks.
+// This is a simple implementation to mimic the actual runtime checks at
+// https://github.com/envoyproxy/envoy/blob/master/source/common/runtime/runtime_impl.cc#L250
```

Comment:
This link will go out of date. If you want to link to other code please use a stable link.